scale-tests/deploy-dispatchers.py

```diff
@@ -167,30 +254,52 @@ def get_default_options(arguments: dict) -> dict:
     return options
 
 
+def get_quota_options(arguments: typing.Dict) -> typing.Dict:
+    """
+    Move the quota options from the command line arguments to a dict.
+    """
+    create_quotas = ast.literal_eval(arguments.get("--create-quotas", True))
+    if not create_quotas:
+        return {}
+
+    resources = ["cpus", "mem", "gpus", ]
+    targets = ["drivers", "executors", ]
+
+    quota_options = {}
+    for t in targets:
+        quota_options[t] = {}
+        for r in resources:
+            arg_key = "--quota-{}-{}".format(t, r)
+            if arg_key in arguments:
+                quota_options[t][r] = arguments[arg_key]
+
+    return quota_options
+
+
 if __name__ == "__main__":
     arguments = docopt(__doc__, version="deploy-dispatchers.py 0.0")
 
     options_file = arguments['--options-json']
     if options_file:
         if not os.path.isfile(options_file):
             # TODO: Replace with logging
-            print("The specified file does not exist: %s", options_file)
+            log.error("The specified file does not exist: %s", options_file)
             sys.exit(1)
 
         options = json.load(open(options_file, 'r'))
     else:
         options = get_default_options(arguments)
 
+
+    if arguments['--package-repo']:
+        sdk_repository.add_stub_universe_urls([arguments['--package-repo']])
+
+    quota_options = get_quota_options(arguments)
+
     deploy_dispatchers(
         num_dispatchers=int(arguments['<num_dispatchers>']),
         service_name_base=arguments['<service_name_base>'],
         output_file=arguments['<output_file>'],
+        linux_user=arguments["--user"],
```

Comment:
Is the user needed? Or is this a *different* user?

Reply:
I assume this is the user that the dispatcher task runs as.

Reply:
Yes, that is how I understood it.